src/text_statistics/TextStatistics.java

```diff
@@ -0,0 +1,66 @@
+package text_statistics;
+
+public class TextStatistics {
+    public static int countCharsWithSpaces(String fromUser){
+        int count = 0;
+        for (int i = 0; i < fromUser.length(); i++) {
+            if (Character.isLetter(fromUser.charAt(i)))
+                count++;
+            if (!Character.isLetter(fromUser.charAt(i)))
+                count++;
+        }
+        return count;
+    }
+    public static int countCharsWithoutSpaces(String fromUser){
+        int count = 0;
+        for(int i = 0; i < fromUser.length(); i++) {
+            if(fromUser.charAt(i) != ' ')
+                count++;
+        }
+
+        return count;
+    }
+    public static int countWords(String fromUser) {
+        if (fromUser == null || fromUser.isEmpty()) {
```

Comment:
Tutaj też można było to zrobić w ten sposób `text.split(" ").length` - rozdzielasz całe zdanie w miejscach gdzie masz spacje (jest to granica pomiędzy słowami) to zwraca Ci tablicję słów i sprawdzasz jaka jest jej długość. Odpowiada to ilości słów w tekście